tests/installation/reboot_after_installation.pm

```diff
@@ -27,7 +28,7 @@ sub run {
         $svirt->change_domain_element(os => boot => {dev => 'hd'});
     }
     # Reboot
-    if (is_remote_backend) {
+    if (is_remote_backend && !is_svirt_except_s390x) {
```

Comment:
hm, it get's the job done but I wonder if we can not find a better way to describe it to work with a single condition that is obvious to everyone. So what makes s390x in common with e.g. ipmi and spvm but not as generic as `is_remote_backend`? Can we find a proper method name to explain what we *actually* need here? Do you have an idea?

Reply:
Or does this only apply to physical machines?

Reply:
hm, I guess in principle this if should apply for all backends where the console used for installation is provided by the SUT directly and not the hypervisor. Sounds more like we want to use `has_ttys` from Utils::Backends here. @mnowaksuse does this make sense?

Reply:
@okurz Thanks for pointing out `has_ttys()`, I think that's the right ingredient. And for the actual question: no, sorry, no idea.